pingraham/summit_notebooks/AT_20210305/daytime_tests_20210308.ipynb

```diff
@@ -0,0 +1,1463 @@
+      "Read 1 history items for RemoteEvent(ATPtg, 0, ptgRotCurrentWrap)\n",
+      "Read 1 history items for RemoteEvent(ATPtg, 0, rotWrapWarning)\n",
+      "Read 100 history items for RemoteEvent(ATPtg, 0, summaryState)\n",
+      "Read 1 history items for RemoteEvent(ATPtg, 0, sunProximityWarning)\n",
+      "Read 100 history items for RemoteEvent(ATPtg, 0, timesOfLimits)\n",
+      "Read 100 history items for RemoteEvent(ATPtg, 0, trackPosting)\n",
+      "Read 64 history items for RemoteEvent(ATPtg, 0, wavelength)\n",
+      "Read 100 history items for RemoteEvent(ATPtg, 0, weatherDataApplied)\n",
+      "Read 100 history items for RemoteEvent(ATPtg, 0, weatherDataInvalid)\n",
+      "Read historical data in 0.44 sec\n",
+      "Read 8 history items for RemoteEvent(ATCamera, 0, appliedSettingsMatchStart)\n",
+      "Read 100 history items for RemoteEvent(ATCamera, 0, ccsCommandState)\n",
+      "Read 100 history items for RemoteEvent(ATCamera, 0, endOfImageTelemetry)\n",
+      "Read 100 history items for RemoteEvent(ATCamera, 0, endReadout)\n",
+      "Read 100 history items for RemoteEvent(ATCamera, 0, endShutterClose)\n",
+      "Read 100 history items for RemoteEvent(ATCamera, 0, endShutterOpen)\n",
+      "Read 10 history items for RemoteEvent(ATCamera, 0, focalPlaneHardwareIdSettingsApplied)\n",
+      "Read 10 history items for RemoteEvent(ATCamera, 0, focalPlaneSummaryInfo)\n",
+      "Read 100 history items for RemoteEvent(ATCamera, 0, heartbeat)\n",
+      "Read 100 history items for RemoteEvent(ATCamera, 0, imageReadinessDetailedState)\n",
+      "Read 100 history items for RemoteEvent(ATCamera, 0, imageReadoutParameters)\n",
+      "Read 100 history items for RemoteEvent(ATCamera, 0, raftsDetailedState)\n",
+      "Read 8 history items for RemoteEvent(ATCamera, 0, settingVersions)\n",
+      "Read 100 history items for RemoteEvent(ATCamera, 0, shutterDetailedState)\n",
+      "Read 100 history items for RemoteEvent(ATCamera, 0, shutterMotionProfile)\n",
+      "Read 8 history items for RemoteEvent(ATCamera, 0, softwareVersions)\n",
+      "Read 100 history items for RemoteEvent(ATCamera, 0, startIntegration)\n",
+      "Read 100 history items for RemoteEvent(ATCamera, 0, startReadout)\n",
+      "Read 32 history items for RemoteEvent(ATCamera, 0, summaryState)\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "[[None, None, None, None, None, None, None], [None, None, None, None]]"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "timeAndDate DDS read queue is full (100 elements); data may be lost\n",
+      "mount_positions DDS read queue is full (100 elements); data may be lost\n"
+     ]
+    }
+   ],
+   "source": [
+    "#get classes and start them\n",
+    "domain = salobj.Domain()\n",
```

Comment:
I usually recommend splitting the definition of the `domain` and the classes (`ATCS` etc) into different cells...